satpy/readers/ahi_hsd.py

```diff
@@ -775,3 +743,96 @@ def _ir_calibrate(self, data):
         c2_ = self._header["calibration"]["c2_rad2tb_conversion"][0]
 
         return (c0_ + c1_ * Te_ + c2_ * Te_ ** 2).clip(0)
+
+
+class NominalTimeCalculator:
```

Comment:
Can we make this private so we don't have to worry about someone trying to use it outside this module?

```suggestion
class _NominalTimeCalculator:
```

Reply:
Good idea, done